charts/devlake/values.yaml

```diff
@@ -223,3 +226,14 @@ option:
 
   # database type, supported: [mysql]
   database: mysql
+
+  # if true, will not add the database connection details 
+  # to the configmap and will let them be obtained from a secret.
+  useConnectionDetailsSecret: false
```

Comment:
It would be great if a screenshot of a local test without secrets was provided, to show that backward compatibility (without secrets) is ensured.

Reply:
![Screenshot 2023-04-27 at 10 14 08](https://user-images.githubusercontent.com/5280729/234801827-84d36ac0-bc5d-42ac-95c8-3ba4544f3795.png)